lib/v2/guggenheim/exhibitions.js

```diff
@@ -0,0 +1,35 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const vm = require('vm');
+
+function parseDate(dateObj) {
+    return Date.parse(`${dateObj.month} ${dateObj.day}, ${dateObj.year}`);
+}
```

Comment:
```suggestion
const { parseDate} = require('@/utils/parse-date');
```
docs: https://docs-rsshub.pages.dev/en/joinus/pub-date.html